### PR TITLE
Blog added to TodoList

### PR DESCRIPTION
**Pull request by mabuhanifa**
Link blog added to and blog route create and added to TodoList

### CODE REVIEW

File: src/components/Blog.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            <p>
+              React is a popular JavaScript library for building user
+              interfaces. It uses a component-based architecture, allowing
+              developers to create reusable UI elements. React virtual DOM
+              efficiently updates only the necessary parts of the user
+              interface, resulting in fast and responsive applications. Its
+              ecosystem offers a wide range of libraries and tools for state
+              management, routing, and testing, making it versatile for building
+              complex web applications.
+            </p>
```
Comment (abuhanifa96) [inline src/components/Blog.tsx:19]: put a title top of paragraph

**Review by abuhanifa96 — COMMENTED** [review]
make a title for every paragraph

File: src/components/Blog.tsx (hunk 1 above)
Comment (mabuhanifa) [reply src/components/Blog.tsx:19]: ok

**Review by mabuhanifa — COMMENTED** [review]
adding title

**Pull request merged**